core/consensus/aura/src/lib.rs

```diff
@@ -272,11 +269,13 @@ pub fn start_aura2<B, C, E, I, P, SO, Error, OnExit>(
 		sync_oracle,
 		on_exit,
 		inherent_data_providers,
-		accept_old_seals,
+		false,
 	)
 }
 
 /// Start the aura worker. The returned future should be run in a tokio runtime.
+///
+/// This version accepts old-style seals.
```

Comment:
I would like it to be that accepting old seals is opt-in, since only chains that have to worry about backwards compatibility should accept old-style seals. This API makes it a bit harder to phase out.